File: app/src/main/java/us/shiroyama/wireless/DeviceViewHolder.java
package us.shiroyama.wireless;

import android.view.View;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

public class DeviceViewHolder extends RecyclerView.ViewHolder {
    public final TextView textViewDevice;
    public final TextView textViewAddress;
    public final TextView textViewType;

    public DeviceViewHolder(@NonNull View itemView) {
        super(itemView);
        textViewDevice = itemView.findViewById(R.id.textViewDevice);
        textViewAddress = itemView.findViewById(R.id.textViewAddress);
        textViewType = itemView.findViewById(R.id.textViewType);
    }
}

File: app/src/main/java/us/shiroyama/wireless/DeviceViewHolder.kt
package us.shiroyama.wireless

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

class DeviceViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @JvmField
    val textViewDevice: TextView

    @JvmField
    val textViewAddress: TextView

    @JvmField
    val textViewType: TextView

    init {
        textViewDevice = itemView.findViewById(R.id.textViewDevice)
        textViewAddress = itemView.findViewById(R.id.textViewAddress)
        textViewType = itemView.findViewById(R.id.textViewType)
    }
}